apps/browser/src/platform/system-notifications/browser-system-notification.service.ts

```diff
@@ -0,0 +1,109 @@
+import { Observable, Subject } from "rxjs";
+
+import { DeviceType } from "@bitwarden/common/enums";
+import { LogService } from "@bitwarden/common/platform/abstractions/log.service";
+import { PlatformUtilsService } from "@bitwarden/common/platform/abstractions/platform-utils.service";
+import {
+  ButtonLocation,
+  SystemNotificationClearInfo,
+  SystemNotificationCreateInfo,
+  SystemNotificationEvent,
+  SystemNotificationsService,
+} from "@bitwarden/common/platform/notifications/system-notifications-service";
+
+export class BrowserSystemNotificationService implements SystemNotificationsService {
+  private systemNotificationClickedSubject = new Subject<SystemNotificationEvent>();
+  notificationClicked$: Observable<SystemNotificationEvent>;
+
+  constructor(
+    private logService: LogService,
+    private platformUtilsService: PlatformUtilsService,
+  ) {
+    this.notificationClicked$ = this.systemNotificationClickedSubject.asObservable();
```

Comment:
We are going to need to make this observable be a listener on the notifications directly instead of through the subject. It's actually very important that the listeners are added during the service worker startup. 

```suggestion
    this.notificationClicked$ = merge([
        fromChromeEvent(chrome.notifications.onButtonClicked).pipe(
            map(...),
        ),
        fromChromeEvent(chrome.notifications.onClicked).pipe(
            map(...),
        ),
    ]);
```